kubejs/server_scripts/force_reload.js

```diff
@@ -0,0 +1,3 @@
+ServerEvents.loaded((event) => {
+    event.server.runCommandSilent('reload');
```

Comment:
what's going on here? 😄

Reply:
That's the 'fix' for tags blowing up. Just one quick reload as the server starts. It's been almost imperceptible so far

Reply:
Cool 👍